Makefile.pre.in

```diff
@@ -757,6 +757,7 @@ Modules/getpath.o: $(srcdir)/Modules/getpath.c Makefile
 		-DEXEC_PREFIX='"$(exec_prefix)"' \
 		-DVERSION='"$(VERSION)"' \
 		-DVPATH='"$(VPATH)"' \
+		-DEXE_SUFFIX='"$(EXE)"' \
```

Comment:
This is not needed: suffix is always `.exe`.

Reply:
Indeed, now that is true.